record.c

```diff
@@ -384,17 +416,33 @@ int janus_recorder_save_frame(janus_recorder *recorder, char *buffer, uint lengt
 				res, sizeof(gint64), g_strerror(errno));
 		}
 	}
+	/* Edit packet header if needed */
+	rtp_header *header = (rtp_header *)buffer;
```

Comment:
`janus_rtp_header`: `rtp_header` is only there for backwards compatibility, as that's how we called it originally, which we want to get rid of.